stdlib/2/subprocess.pyi

```diff
@@ -4,7 +4,7 @@
 
 from typing import Sequence, Any, Mapping, Callable, Tuple, IO, Union, Optional, List, Text
 
-_FILE = Union[int, IO[Any]]
+_FILE = Optional[Union[int, IO[Any]]]
```

Comment:
Is there a performance reason or anything other than style for this decision (I'm just interested - Is this a pyi file thing)? I find Optional way more explicit that it can have the optional 'None' Value, since we're told to use that else where.